cluster/gce/gci/configure-helper.sh

```diff
@@ -794,6 +794,17 @@ function compute-master-manifest-variables {
   fi
 }
 
+# A helper function that bind mounts kubelet dirs for running mount in a chroot
+function bind-mount-kubelet {
+  local -r CONTAINERIZED_MOUNTS_HOME="${KUBE_HOME}/mounter-rootfs"
+  mount --bind "${CONTAINERIZED_MOUNTS_HOME}" "${CONTAINERIZED_MOUNTS_HOME}"
+  mount -o remount,exec "${CONTAINERIZED_MOUNTS_HOME}"
+  mount --rbind /var/lib/kubelet/ "${CONTAINERIZED_MOUNTS_HOME}/var/lib/kubelet"
+  mount --make-rshared "${CONTAINERIZED_MOUNTS_HOME}/var/lib/kubelet"
+  mount --rbind /proc "${CONTAINERIZED_MOUNTS_HOME}/proc"
+  mount --rbind /dev "${CONTAINERIZED_MOUNTS_HOME}/dev"
```

Comment:
from a recent issue investigation, I think we have to bind mount /etc and /var here